src/modules/Game.class.js

```diff
@@ -20,25 +16,256 @@ class Game {
    * If passed, the board will be initialized with the provided
    * initial state.
    */
-  constructor(initialState) {
-    // eslint-disable-next-line no-console
-    console.log(initialState);
+  constructor(
+    cells = [
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+      [0, 0, 0, 0],
+    ],
+  ) {
+    this.cells = cells;
+    this.restart();
   }
 
-  moveLeft() {}
-  moveRight() {}
-  moveUp() {}
-  moveDown() {}
+  moveLeft() {
+    if (this.status !== 'playing') {
+      return;
+    }
+
+    let moved = false;
+
+    for (let i = 0; i < 4; i++) {
+      const row = this.board[i].filter((val) => val !== 0);
```

Comment:
The `moveLeft` method uses `this.board`, but the constructor initializes `this.cells`. Ensure that `this.board` is initialized correctly before using it in game logic.